homeassistant/components/myq/light.py

```diff
@@ -114,7 +103,7 @@ def device_info(self):
             "sw_version": self._device.firmware_version,
         }
         model = KNOWN_MODELS.get(self._device.device_id[2:4])
```

Comment:
You should remove this line :)

Reply:
Oh shoot. I was more looking into what the := did as I had not seen that yet that I totally forgot about removing that line then. 
Just removed and pushed the commit.